src/controllers/ContactController.php

```diff
@@ -175,13 +175,15 @@ public function actionAttachDocuments($id)
         $model = new DocumentUploadForm(['id' => $contact->id]);
         if ($model->load(Yii::$app->request->post()) && $model->validate()) {
             $session = Yii::$app->session;
-            if ($model->save()) {
+            try {
+                if (!$model->save()) {
+                    throw new \RuntimeException($model->getFirstError('title'));
+                }
                 $session->addFlash('success', Yii::t('hipanel:client', 'Documents were saved'));
-
                 return $this->redirect(['attach-documents', 'id' => $id]);
+            } catch (\Throwable $e) {
```

Comment:
```suggestion
            } catch (Throwable $e) {
```

Старайся такое импортировать, пожалуйста